### PR TITLE
Update Seahorse to 42

### PR DESCRIPTION
**Pull request by bbhtt**

### CODE REVIEW

**Comment by bbhtt** [comment]
The default branch was changed from master to main but the screenshot tag in appdata still points to the old branch. Didn't notice this in the local build which completed successfully,  guess it doesn't try to download the screenshot.

Adding a patch

File: 0001-Fix-screenshot-location.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+diff --git a/data/org.gnome.seahorse.Application.appdata.xml.in b/data/org.gnome.seahorse.Application.appdata.xml.in
+index 2217d86b..5ad4abaf 100644
+--- a/data/org.gnome.seahorse.Application.appdata.xml.in
++++ b/data/org.gnome.seahorse.Application.appdata.xml.in
+@@ -18,7 +18,7 @@
+ 
+   <screenshots>
+     <screenshot type="default">
+-      <image width="1024" height="576">https://gitlab.gnome.org/GNOME/seahorse/raw/master/data/screenshot-default.png</image>
++      <image width="1024" height="576">https://gitlab.gnome.org/GNOME/seahorse/raw/main/data/screenshot-default.png</image>
```
Comment (nielsdg) [inline 0001-Fix-screenshot-location.patch:19]:
```suggestion
+      <image width="1024" height="576">https://gitlab.gnome.org/GNOME/seahorse/raw/gnome-42/data/screenshot-default.png</image>
```

Comment (nielsdg) [reply 0001-Fix-screenshot-location.patch:19]: Thanks for finding this by the way!

**Comment by bbhtt** [comment]
The build was fine, looks like uploading the build for x86_64 failed somehow

`Api call to https://hub.flathub.org/api/v1/build/90683/build_ref failed with status 500, details: {'status': 500, 'error-type': 'no-error', 'message': 'No json error details from server'}`

**Comment by nielsdg** [comment]
Agreed, LGTM. Thanks! :-)

**Pull request merged**